webapp/client/src/pages/FilingSuccessPage.js

```diff
@@ -0,0 +1,68 @@
+import PropTypes from "prop-types";
+import React, { useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import StepHeaderButtons from "../components/StepHeaderButtons";
+import FormHeader from "../components/FormHeader";
+import DownloadLink from "../components/DownloadLink";
+import StepNavButtons from "../components/StepNavButtons";
+
+export default function FilingSuccessPage({
+  nextUrl,
+  transferTicket,
+  downloadUrl,
+  taxNumberProvided,
+  plausibleDomain,
+}) {
+  const { t } = useTranslation();
+  const stepHeader = {
+    title: t("filing.success.title"),
+    intro: t("filing.success.intro"),
+  };
+
+  // Add custom plausible goal if tax number provided and plausible domain active
+  // Hook is called after component is rendered and will only run once after component is rendered
+  useEffect(() => {
+    if (taxNumberProvided !== null && plausibleDomain !== null) {
+      const addPlausibleGoal = () => {
+        window.plausible("summary_submitted", {
+          props: { tax_number_provided: { taxNumberProvided } },
+        });
+      };
+      document.addEventListener("DOMContentLoaded", addPlausibleGoal);
+      // remove listener once the component unmounts
+      return () => {
+        document.removeEventListener("DOMContentLoaded", addPlausibleGoal);
+      };
```

Comment:
UseEffect is called once. Could we get rid of the eventListener then and call addPlausibleGoal directly? 

BTW: We can leave that here for now but if we set more custom goals that should be a function independently of this component.

Reply:
Yes that is possible i think: getting rid of the eventListener and calling the custom goal within the useEffect, I will change it but leave it for now there?

Reply:
Yes, that's fine with me